src/types.js

```diff
@@ -1,13 +1,13 @@
 // @flow
 
-export type PrepackConfigurationType = Object;
+export type PrepackConfigType = Object;
 
-export type PluginConfigurationType = {
-  prepack: PrepackConfigurationType,
+export type PluginConfigType = {
+  prepack: PrepackConfigType,
   test: RegExp
 };
 
-export type UserPluginConfigurationType = {
```

Comment:
Please don't start changing variable names in project based on personal preference.

The convention is to use full names, regardless of how long.

Reply:
Fixed in a61d42b